src/Gren/Kernel/Crypto.js

```diff
@@ -0,0 +1,660 @@
+/*
+
+import Gren.Kernel.Scheduler exposing (binding, succeed, fail)
+import Gren.Kernel.Bytes exposing (writeBytes)
+import Crypto exposing (RsaSsaPkcs1V1_5SigningError, RsaPssSigningError, AesCtrEncryptionError, RsaOaepEncryptionError, P256, P384, P521, AesLength128, AesLength192, AesLength256, CanBeExtracted, CannotBeExtracted, HmacKey, Sha256, Sha384, Sha512, SignWithRsaPssError, AesGcmDecryptionError, AesGcmEncryptionError, AesCbcDecryptionError, AesCbcEncryptionError, AesCtrDecryptionError, DecryptWithRsaOaepError, ImportRsaKeyError, ImportHmacKeyError, ImportEcKeyError, ImportAesKeyError, Key, SecureContext, PublicKey, PrivateKey)
+import Maybe exposing (Just, Nothing)
+import Bytes exposing (Bytes)
+
+*/
+
+var crypto = function () {
```

Comment:
I think you should rename this to `_Crypto_impl`, or some other namespaced name. All kernel files live in the same namespace, so if any other current or future kernel module defines a `crypto` variable, it'll mess things up.